debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -66,11 +70,22 @@ public class RedisStreamChangeConsumer extends BaseChangeConsumer
 
     private RedisStreamChangeConsumerConfig config;
 
+    private String heartbeatPrefix;
+
     @PostConstruct
     void connect() {
-        Configuration configuration = Configuration.from(getConfigSubset(ConfigProvider.getConfig(), ""));
+        // Get configuration from ConfigProvider
+        Config mpConfig = ConfigProvider.getConfig();
+        Map<String, Object> sourceConfig = getConfigSubset(mpConfig, "debezium.source.");
+
+        // Get Redis sink configuration
+        Configuration configuration = Configuration.from(getConfigSubset(mpConfig, ""));
```

Comment:
```suggestion
        Configuration sinkConfig = Configuration.from(getConfigSubset(mpConfig, ""));
```

Reply:
I don't think renaming to `sinkConfig` would be accurate. The variable contains all configuration properties, not just sink-specific ones. The name `configuration` is actually more appropriate here since it represents the complete set of properties, IMO.
Renaming it to `sinkConfig` would be misleading because it implies the configuration only contains sink-related properties, which isn't the case.

@jpechane WDYT?